index.bs

```diff
@@ -59,9 +74,8 @@ dictionary CSSStyleSheetInit {
 			otherwise,
 			set <var>sheet’s</var> CSS rules to an empty list.
 		6. Let <var>promise</var> be a promise.
-		7. Wait for loading of <a spec=css-cascade-4>@import</a> rules in <var>sheet</var>.
-			* If any of them resulted in a resource with a <a spec=html>Content-Type metadata</a> of anything other than <code>text/css</code>, reject <var>promise</var> with a {{TypeError}}.
-			* If any of them failed to load, reject <var>promise</var> with reason set to {{DOMException}} with the message "Loading import rules failed".
+		7. Wait for loading of <a spec=css-cascade-4>@import</a> rules in <var>sheet</var> and any nested loads from those rules in parallel.
+			* If any of them failed to load or resulted in a resource with a <a spec=html>Content-Type metadata</a> of anything other than <code>text/css</code>, reject <var>promise</var> with reason set to {{DOMException}} with name {{NotAllowedError}}.
```

Comment:
Here and elsewhere, you can use the more conventional `"{{NotAllowedError}}" {{DOMException}}`.